styles/_components.scss

```diff
@@ -2,4 +2,5 @@
 @import './components/buttonWhiteWithIcon';
 @import './components/logo';
 @import './components/visualImage';
+@import './components/menuOptions';
```

Comment:
Your scss file starts with capital case letter, consider change to lower case letter, so it will pass GH checks.